### PR TITLE
Ecs taskdefinition secrets

### PR DESCRIPTION
**Pull request by cristidas**
##### SUMMARY
Updating the documentation for ecs_taskdefinition module to include example on how to include in the container definition environment variables and secrets from Parameter Store or Secrets Manager.

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
ecs_taskdefinition

### CODE REVIEW

**Comment by cristidas** [comment]
resolves #55031

File: lib/ansible/modules/cloud/amazon/ecs_taskdefinition.py [hunk 1]
```diff
@@ -173,6 +173,28 @@
...
+    containers:
+    - name: nginx
+      essential: true
+      image: "nginx"
+      environment:
+        - name: "PORT"
+          value: "8080"
+      secrets:
+        - name: "VAR_NAME_1"
+          valueFrom: "arn:aws:secretsmanager:region:aws_account_id:secret:secret_name_for_ENV_NAME_1"
```
Comment (willthames) [inline lib/ansible/modules/cloud/amazon/ecs_taskdefinition.py]: probably clearer as `secret_name_for_VAR_NAME_1` (same for `VAR_NAME_2` below)

**Review by willthames — CHANGES_REQUESTED** [review]
Overall in favour of this once the `valueFrom` is slightly clearer

File: lib/ansible/modules/cloud/amazon/ecs_taskdefinition.py (hunk 1 above)
Comment (cristidas) [reply lib/ansible/modules/cloud/amazon/ecs_taskdefinition.py]: Thanks @willthames. Changes committed.

**Comment by cristidas** [comment]
ready_for_review

**Comment by acozine** [comment]
Thanks @cristidas for picking up that issue and creating a PR to address it! @willthames I think your comments have been addressed - if you're not happy with the final version, ping me on IRC.

**Pull request merged**